test/hotspot/jtreg/runtime/Monitor/libMonitorWithDeadObjectTest.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
nit: why include 2022 in the copyright header?

Reply:
Because the test was copied and built upon a test that was created in 2022.